src/components/Splash/Splash.jsx

```diff
@@ -72,23 +73,19 @@ const Splash = () => {
 
             { showSponsors ? (
               <React.Suspense fallback={<SponsorsPlaceholder />}>
-                <h2>Latest Sponsors</h2>
-                <Support rank="latest" />
+                <p><input type="checkbox" checked={supportType === 'monthly'} onChange={e => setSupportType(e.target.checked ? 'monthly' : 'total')} /> Show sponsors by their average monthly amount of sponsoring in the last year.</p>
```

Comment:
```suggestion
                <p><label><input type="checkbox" checked={supportType === 'monthly'} onChange={e => setSupportType(e.target.checked ? 'monthly' : 'total')} /> Show sponsors by their average monthly amount of sponsoring in the last year.</label></p>
```

Reply:
We can wrap the text in a `label` tag so user can also click the text to toggle, currently we have to click the small checkbox.